third-party/brotli/enc/hash_composite_inc.h

```diff
@@ -43,7 +43,7 @@ static void FN(Initialize)(
   self->ha = 0;
   self->hb = 0;
   self->params = params;
-  /* TODO: Initialize of the hashers is defered to Prepare (and params
+  /* TODO: Initialize of the hashers is deferred to Prepare (and params
```

Comment:
Don't patch anything in third-party. These sources are imported unchanged from upstream sources so any local changes just make our lives difficult.